scripts/seed-mock-data.sh

```
@@ -0,0 +1,37 @@
+# pnpm seed
+
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)